src/index.html

```diff
@@ -64,7 +64,7 @@ <h2>{{ "TITLE_DOWNLOAD_WALLET" | translate }}</h2>
                             <input type="password" ng-model="password" required>
                         </md-input-container>
                         <md-input-container class="md-block" flex>
-                            <md-button type="submit" ng-disabled="downloadForm.$invalid">{{ "BUTTON_DOWNLOAD_WALLET" | translate }}</md-button>
+                            <md-button type="submit" ng-disabled="downloadForm.$invalid">{{ BUTTON_DOWNLOAD_WALLET | translate }}</md-button>
```

Comment:
Could you revert this change?

Reply:
I was afraid of revert's mistake, so I added "".
Is it not okay?